lib/tasks/pkg.rake

```diff
@@ -24,5 +24,20 @@ namespace :pkg do
     version = `git show #{ref}:VERSION`.chomp.chomp('-develop')
     raise "can't find VERSION from #{ref}" if version.length == 0
     `git archive --prefix=foreman-#{version}/ #{ref} | bzip2 -9 > pkg/foreman-#{version}.tar.bz2`
+
+    `npm install`
```

Comment:
This should be run with `--no-optional` flag. Also, possibly precede it with an `npm install npm` to make sure we are using a current npm version, then execute using the installed version.